libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -35,6 +35,8 @@ export class Function extends cloud.FunctionBase {
   private policyStatements?: any[];
   /** Function ARN */
```

Comment:
Please add information in the docstring explaining exactly what's going to be returned from each type of ARN property.